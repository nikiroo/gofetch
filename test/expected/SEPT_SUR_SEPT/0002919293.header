0"18 Temps": le slam étonnant d%27André Flahaut	null/SEPT_SUR_SEPT/0002919293		70
i  [Elections 2018] 26-09-2018 15:50
i
i    %2718  temps%27,  un  %27slam citoyen et engagé%27 dédié %27aux
i    jeunes  (d%27abord) et aux moins jeunes (aussi)%27, écrit André
i    Flahaut en guise de ...
i
