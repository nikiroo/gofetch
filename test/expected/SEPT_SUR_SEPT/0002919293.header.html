<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h1><a href='0002919293.html'>"18 Temps": le slam étonnant d%27André Flahaut</a></h1>
	<div class='details'>([Elections 2018] 26-09-2018 15:50)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		%2718 temps%27, un %27slam citoyen et engagé%27 dédié %27aux jeunes (d%27abord) et aux moins jeunes (aussi)%27, écrit André Flahaut en guise de ...
	</div>
<hr/>
</div>
</body>
